flutter/Graminsta/lib/main.dart

```diff
@@ -30,15 +32,12 @@ class _HomePageState extends State<HomePage> {
   int _selectedIndex = 0;
   static const TextStyle optionStyle =
       TextStyle(fontSize: 30, fontWeight: FontWeight.bold);
-  static const List<Widget> _widgetOptions = <Widget>[
+  static List<Widget> _widgetOptions = <Widget>[
     Text(
       'Index 0: Home',
       style: optionStyle,
     ), //change this into the timeline page.
-    Text(
-      'Index 1: Person',
-      style: optionStyle,
-    ), //change this into the gallery page.
+    UsersList(),
```

Comment:
This page was intended to be gallery, I changed it to show people because I think personal gallery page can be jumped to by clicking the user listed on this users list page.